### DISCUSSION TITLE
What Does 'MS2.Scan' Mean?

### DISCUSSION THREAD

**Discussion by ec-ho-ra-mos** [vdemichev/DiaNN#985] [General]
I understand that DIA-NN is not a spectrum-centric search engine. This means it doesn't rely solely on a single MS2 spectrum for peptide identification. In the DIA-NN output, "RT.Start" and "RT.End" indicate the RT range of the precursor's elution peak. I believe RT refers of the topmost measurement of the peak, and is often around the middle between RT.Start and RT.End

Now, what does "MS2.Scan" signify? Does it represent the MS2 spectrum index corresponding to the spectrum at the "RT" (the topmost part of the elution peak)?

**Comment by vdemichev** [comment]
MS2.Scan is the index of the respective MS2 scan at the apex (RT) of the peptide elution, using DIA-NN's internal indexing of the MS2 scans. For classic DIA/SWATH this is just the number of MS2 scans preceding the scan in question.

**Reply by ec-ho-ra-mos** [reply]
Thank you very much for your prompt response!
